src/tls_wrap.cc

```diff
@@ -375,6 +382,7 @@ Local<Value> TLSWrap::GetSSLError(int status, int* err, const char** msg) {
           *msg = buf;
         }
         static_cast<void>(BIO_reset(bio));
+        BIO_free_all(bio);
```

Comment:
Does that mean that all the BIO_new + BIO_reset calls in src/node_crypto.cc are leaking memory as well?